pwa/src/components/PatientInfoForm.tsx

```diff
@@ -0,0 +1,265 @@
+      />
+
+      <TextareaItem
+        title="Nickname"
+        placeholder="Nickname"
+        id="nickname"
+        autoHeight
+      />
+
+      <List renderHeader={() => "Sex"}>
+        {sexValues.map((i) => (
+          <RadioItem
+            data-testid={`${i.label.toLowerCase()}-button`}
+            id={`${i.label.toLowerCase()}-button`}
+            key={i.value}
+            name="sex"
+            checked={sex === i.value}
+            onClick={() => setSex(i.value)}
+          >
+            {i.label}
+          </RadioItem>
+        ))}
+      </List>
+
+      <WhiteSpace />
+
+      <InputItem placeholder="YYYY-MM-DD" id="birthdate">
+        Birth Date
+      </InputItem>
+
+      <InputItem type="number" placeholder="Age" id="age">
+        Age{" "}
+      </InputItem>
+
+      <InputItem type="number" placeholder="Phone" id="phone">
+        Phone
+      </InputItem>
+
+      <TextareaItem title="Town" placeholder="Town" id="town" autoHeight />
+
+      <List renderHeader={() => "Visit?"}>
+        {radioYesNo.map((i) => (
+          <RadioItem
+            key={i.value}
+            data-testid={`visit-${i.label}`}
+            name="visit"
+            checked={visit === i.value}
+            onClick={() => setVisit(i.value)}
+          >
+            {i.label}
+          </RadioItem>
+        ))}
+      </List>
+
+      {/* Only display the 'female-only' div if the patient is female */}
+      {sex ? (
+        <div id="female-only">
+          <WhiteSpace />
+          <List renderHeader={() => "Pregnant?"}>
+            {radioYesNo.map((i) => (
+              <RadioItem
+                data-testid={`pregnant-${i.label}`}
+                key={i.value}
+                name="pregnant"
+                checked={pregnant === i.value}
+                onClick={() => setPregnant(i.value)}
+              >
+                {i.label}
+              </RadioItem>
+            ))}
+          </List>
+
+          <InputItem title="G" type="number" id="gravida">
+            G
+          </InputItem>
+
+          <InputItem title="P" type="number" id="para">
+            P
+          </InputItem>
+
+          <InputItem title="A" type="number" id="abortus">
+            A
+          </InputItem>
+
+          <TextareaItem title="LMP" id="lmp" />
+
+          <List renderHeader={() => "Wants Planning?"}>
+            {radioYesNo.map((i) => (
+              <RadioItem
+                data-testid={`planning-${i.label}`}
+                key={i.value}
+                name="planning"
+                checked={planning === i.value}
+                onClick={() => setPlanning(i.value)}
+              >
+                {i.label}
+              </RadioItem>
+            ))}
+          </List>
+        </div>
+      ) : null}
+
+      <WhiteSpace />
+
+      <InputItem type="number" placeholder="Weight" id="weight">
+        Weight (kg)
+      </InputItem>
+
+      <InputItem type="number" placeholder="Height" id="height">
```

Comment:
This field looks fine on mobile, but on desktop the last parenthesis is getting cut off. 
![Screenshot 2020-04-18 14 33 57](https://user-images.githubusercontent.com/40620013/79669541-e15db300-8181-11ea-959f-9dc5b83fd9c7.png)

Reply:
I was only able to replicate this by zooming out and shrinking my browser. The space between the label and the placeholder is determined by the UI framework, which seems to be a bit off? Maybe due to character spacing between languages or something, don't know.

[This commit](https://github.com/Haiti-Distributed-Healthcare-System/hdhs/pull/66/commits/e1614724978874dff26f2dbecc2b42d0434c8621) enforces additional spacing to avoid unique cases like this. It's a global rule, and due to the framework override, it's marked with `!important`. I'd be against this if it was embedded, but it's a single, non nested, global rule, so I'd like to suggest it.

If people don't like it feel free to sound off here, I'd undo the rule in an additional commit.

If you pull down that change is it fixed for you @abpwrs ?

Reply:
This fixed what I was seeing